packages/lib/components/shared/config/shouldShowMissingPasswordWarning.ts

```diff
@@ -0,0 +1,13 @@
+import SyncTargetRegistry from '../../../SyncTargetRegistry';
+
+const shouldShowMissingPasswordWarning = (syncTargetId: number, settings: any) => {
```

Comment:
This file was added to `lib/components/shared/`. It's possible that there is a better place for this function.

Reply:
This is fine